quinn-proto/src/lib.rs

```diff
@@ -212,7 +212,7 @@ impl fmt::Display for Dir {
 
 /// Identifier for a stream within a particular connection
 #[cfg_attr(feature = "arbitrary", derive(Arbitrary))]
-#[derive(Debug, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
+#[derive(Debug, Default, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
```

Comment:
I'd prefer for this not to have a `Default` impl, instead creating it explicitly in the `StreamMeta::default()` impl (ideally with some comment explaining how we make sure this value never gets used).

Reply:
Ok. I don't think it hurts, since a Stream ID of 0 is just fine and makes sense too. But I am also fine with removing it.